src/solver/simulation/sim_calcul_economique.cpp

```diff
@@ -460,7 +458,7 @@ void SIM_RenseignementProblemeHebdo(const Study& study,
                 problem.CaracteristiquesHydrauliques[k].WeeklyWaterValueStateRegular
                   = getWaterValue(nivInit * 100 / area.hydro.reservoirCapacity,
                                   area.hydro.waterValues,
-                                  weekFirstDay);
+                                  weekFirstDay + 6);
```

Comment:
Please used named variable instead of magic number "6".
If Variable name isn't enough add a comment why "6"